src/main/java/com/google/maps/model/Rankby.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.google.maps.model;
+
+import com.google.maps.internal.StringJoin;
+
+public enum Rankby implements StringJoin.UrlValue {
```

Comment:
Javadoc. Also, it's a nit, but since these things are hard to rename later, should this be RankBy?